### PR TITLE
Added APYRewards and Updated Pool TVLs

### PR DESCRIPTION
**Pull request by byyyyyyyyy**

### CODE REVIEW

**Comment by llamatester** [comment]
The wise-lending-v2 adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       46 passed, 46 total
    Snapshots:   0 total
    Time:        0.277 s
    Ran all test suites.
    
    Nb of pools: 6
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────┬────────────┬───────────────────┬──────────┬─────────────┬────────────────────┬────────────────────┬───────────┬────────────────────────────────────────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────┬────────────────────────────┬────────────────────┬────────────────────┬────────────────────┐
    │ (index) │                                 pool                                  │   chain    │      project      │  symbol  │  poolMeta   │       tvlUsd       │      apyBase       │ apyReward │                                          rewardTokens                                          │                 underlyingTokens                 │            url             │   apyBaseBorrow    │   totalSupplyUsd   │   totalBorrowUsd   │
    ├─────────┼───────────────────────────────────────────────────────────────────────┼────────────┼───────────────────┼──────────┼─────────────┼────────────────────┼────────────────────┼───────────┼────────────────────────────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────┼────────────────────────────┼────────────────────┼────────────────────┼────────────────────┤
    │    0    │ '0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'WETH'  │ 'aArbWETH'  │ 187178.3924274254  │ 4.090518614626984  │     3     │ [ '0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1' ] │ 'https://wiselending.com/' │ 5.170813023783428  │ 187178.3924274254  │ 165635.84668456842 │
    │    1    │ '0x724dc807b04555b71ed48a6896b6F41593b8C637-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'USDC'  │ 'aArbUSDCn' │   100037.011374    │ 7.839496438826336  │     8     │ [ '0x724dc807b04555b71ed48a6896b6F41593b8C637', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0xaf88d065e77c8cC2239327C5EDb3A432268e5831' ] │ 'https://wiselending.com/' │  9.00634190217486  │   100037.011374    │    84501.619628    │
    │    2    │ '0x5979D7b546E38E414F7E9822514be443A4800529-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │ 'wstETH' │  'wstETH'   │ 18745.284281075503 │ 0.0163041029288218 │ undefined │ [ '0x5979D7b546E38E414F7E9822514be443A4800529', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0x5979D7b546E38E414F7E9822514be443A4800529' ] │ 'https://wiselending.com/' │ 0.2301104839608251 │ 18745.284281075503 │ 4675.374609493775  │
    │    3    │ '0x6ab707Aca953eDAeFBc4fD23bA73294241490620-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'USDT'  │ 'aArbUSDT'  │    14960.10851     │ 8.159050042439755  │     8     │ [ '0x6ab707Aca953eDAeFBc4fD23bA73294241490620', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9' ] │ 'https://wiselending.com/' │ 9.632673709816736  │    14960.10851     │    12762.923762    │
    │    4    │ '0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'DAI'   │  'aArbDAI'  │ 14765.358575299771 │ 9.451749347528597  │     8     │ [ '0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1' ] │ 'https://wiselending.com/' │ 10.679929681944218 │ 14765.358575299771 │ 12806.743633641792 │
    │    5    │ '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2-wise-lending-v2-ethereum' │ 'Ethereum' │ 'wise-lending-v2' │  'WETH'  │   'WETH'    │ 515.0931239608324  │         0          │     3     │ [ '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2', '0x78190e4c7C7B2c2C3b0562F1f155a1FC2F5160CA' ] │ [ '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1' ] │ 'https://wiselending.com/' │ 1.2276216283700796 │ 515.0931239608324  │ 216.25172481536976 │
    └─────────┴───────────────────────────────────────────────────────────────────────┴────────────┴───────────────────┴──────────┴─────────────┴────────────────────┴────────────────────┴───────────┴────────────────────────────────────────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────┴────────────────────────────┴────────────────────┴────────────────────┴────────────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by llamatester** [comment]
The wise-lending-v2 adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       46 passed, 46 total
    Snapshots:   0 total
    Time:        0.28 s
    Ran all test suites.
    
    Nb of pools: 6
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────┬────────────┬───────────────────┬──────────┬─────────────┬────────────────────┬────────────────────┬───────────┬────────────────────────────────────────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────┬────────────────────────────┬────────────────────┬────────────────────┬────────────────────┐
    │ (index) │                                 pool                                  │   chain    │      project      │  symbol  │  poolMeta   │       tvlUsd       │      apyBase       │ apyReward │                                          rewardTokens                                          │                 underlyingTokens                 │            url             │   apyBaseBorrow    │   totalSupplyUsd   │   totalBorrowUsd   │
    ├─────────┼───────────────────────────────────────────────────────────────────────┼────────────┼───────────────────┼──────────┼─────────────┼────────────────────┼────────────────────┼───────────┼────────────────────────────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────┼────────────────────────────┼────────────────────┼────────────────────┼────────────────────┤
    │    0    │ '0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'WETH'  │ 'aArbWETH'  │ 177789.71362565283 │ 3.556793263578896  │     3     │ [ '0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1' ] │ 'https://wiselending.com/' │  4.49966069201696  │ 177789.71362565283 │ 154572.8030951598  │
    │    1    │ '0x724dc807b04555b71ed48a6896b6F41593b8C637-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'USDC'  │ 'aArbUSDCn' │    48081.178982    │ 8.866211447427238  │     8     │ [ '0x724dc807b04555b71ed48a6896b6F41593b8C637', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0xaf88d065e77c8cC2239327C5EDb3A432268e5831' ] │ 'https://wiselending.com/' │ 10.26045284491001  │    48081.178982    │    41452.633715    │
    │    2    │ '0x5979D7b546E38E414F7E9822514be443A4800529-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │ 'wstETH' │  'wstETH'   │ 17775.70668495498  │ 0.0177082627629879 │ undefined │ [ '0x5979D7b546E38E414F7E9822514be443A4800529', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0x5979D7b546E38E414F7E9822514be443A4800529' ] │ 'https://wiselending.com/' │ 0.2301129055840109 │ 17775.70668495498  │ 4433.581477184094  │
    │    3    │ '0x6ab707Aca953eDAeFBc4fD23bA73294241490620-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'USDT'  │ 'aArbUSDT'  │ 14959.708127672926 │ 8.156070800276243  │     8     │ [ '0x6ab707Aca953eDAeFBc4fD23bA73294241490620', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9' ] │ 'https://wiselending.com/' │ 9.633775486387718  │ 14959.708127672926 │ 12762.809913208843 │
    │    4    │ '0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE-wise-lending-v2-arbitrum' │ 'Arbitrum' │ 'wise-lending-v2' │  'DAI'   │  'aArbDAI'  │ 14872.587505335403 │ 8.646715310193818  │     8     │ [ '0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE', '0x4A56DCd67E66102E6a877dE8BF2E903Df5E18978' ] │ [ '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1' ] │ 'https://wiselending.com/' │ 10.19764437921425  │ 14872.587505335403 │ 12813.222075019046 │
    │    5    │ '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2-wise-lending-v2-ethereum' │ 'Ethereum' │ 'wise-lending-v2' │  'WETH'  │   'WETH'    │ 487.9381359924955  │         0          │     3     │ [ '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2', '0x78190e4c7C7B2c2C3b0562F1f155a1FC2F5160CA' ] │ [ '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1' ] │ 'https://wiselending.com/' │ 1.2276216283700796 │ 487.9381359924955  │ 204.8512368019829  │
    └─────────┴───────────────────────────────────────────────────────────────────────┴────────────┴───────────────────┴──────────┴─────────────┴────────────────────┴────────────────────┴───────────┴────────────────────────────────────────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────┴────────────────────────────┴────────────────────┴────────────────────┴────────────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

File: src/adaptors/wise-lending-v2/index.js [hunk 1]
```diff
@@ -126,9 +125,11 @@ const getTokenData = async (chain, token, addresses, contract) => {
       project: projectSlug,
       symbol: token,
       poolMeta: usdPrice.symbol,
-      tvlUsd,
+      tvlUsd: totalSupplyUsd,
```
Comment (slasher125) [inline src/adaptors/wise-lending-v2/index.js:128]: for lending pools we use tvlUsd = available liquidity, so totalSupplyUsd - totalBorrowUsd

Comment (vm06007) [reply src/adaptors/wise-lending-v2/index.js:128]: @slasher125 would that pool be picked up if we have $20k in the pool but $15k is borrowed, hence requirement of tvlUSD > $10k suppose we have $90k supplied and $81k borrowed we want our pools to be listed on this page!
https://defillama.com/protocol/wise-lending-v2#yields

Comment (vm06007) [reply src/adaptors/wise-lending-v2/index.js:128]: @slasher125 I guess you can disregard this PR and just review this one instead: https://github.com/DefiLlama/yield-server/pull/1450

**Comment by vm06007** [comment]
@byyyyyyyyy you can close this PR and check this one instead: https://github.com/DefiLlama/yield-server/pull/1450

File: src/adaptors/wise-lending-v2/index.js (hunk 1 above)
Comment (vm06007) [reply src/adaptors/wise-lending-v2/index.js:128]: @slasher125 I still have a question if those pools that have more than 10k in total supplied would be displayed if users borrowed funds from them utilizing more than 90% of the pool, hence we can have 20k in the pool but 15k is borrowed out... we want these pools to still be visible.

**Pull request closed without merging**